docs/content/patterns/alz/Cleaning-up-a-Deployment.md

```diff
@@ -39,29 +39,29 @@ Follow the instructions below to download the cleanup script file. Alternatively
 3. Change directories to the location of the **Start-AMBACleanup.ps1** script
 4. Configure the _**$pseudoRootManagementGroup**_ variable using the command below:
 
-  ```powershell
-  $pseudoRootManagementGroup = "The pseudo root management group id parenting the identity, management and connectivity management groups"
-  ```
+    ```powershell
+    $pseudoRootManagementGroup = "The pseudo root management group id parenting the identity, management and connectivity management groups"
+    ```
 
 5. Sign in to the Azure with the `Connect-AzAccount` command. The account you sign in as needs to have permissions to remove Policy Assignments, Policy Definitions, and resources at the desired Management Group scope.
 6. Execute the script using one of the options below:
 
     {{% include "PowerShell-ExecutionPolicy.md" %}}
 
-    **Generate a list of the resource IDs which would be deleted by this script:**
+    **Show output of what would happen if deletes executed:**
 
     ```powershell
-    ./Start-AMBACleanup.ps1 -pseudoRootManagementGroup $pseudoRootManagementGroup -ReportOnly
+    ./Start-AMBACleanup.ps1 -pseudoManagementGroup $pseudoRootManagementGroup -WhatIf
```

Comment:
Can you validate? Looks like the param of the script is `[string]$pseudoRootManagementGroup`

Reply:
Done